cmd/sidecar-proxy/main.go

```diff
@@ -127,15 +127,6 @@ func (pi *ProxyInstance) Start(proxyHost, access, refresh string) error {
 			},
 		}
 	}
-	pi.rp.ModifyResponse = func(r *http.Response) error {
```

Comment:
Why are these changes being made in this PR?

Reply:
This was a debug statement that was left erroneously as part of the powerscale feature.